tutorial/markdown/kotlin/ktor.md

```diff
@@ -2,72 +2,65 @@
 # frontmatter
 path: "/tutorial-quickstart-kotlin-ktor"
 # title and description do not need to be added to markdown, start with H2 (##)
-title: Start with Kotlin and Ktor
+title: Quickstart in Couchbase with Kotlin and Ktor
 short_title: Couchbase Kotlin SDK with Ktor
 description: 
-  - Build a REST API with Couchbase's Java SDK 3 and Kotlin
-  - Explore the various features of Ktor
-content_type: tutorial
+  - Learn to build a REST API in Kotlin using Ktor and Couchbase
+  - See how you can fetch data from Couchbase using SQL++ queries
+  - Explore CRUD operations in action with Couchbase
+content_type: quickstart
 filter: sdk
-technology: 
+technology:
   - kv
+  - index
   - query
 tags:
+  - Ktor
   - REST API
 sdk_language:
   - kotlin
 length: 30 Mins
 ---
 
-[![Try it now!](https://da-demo-images.s3.amazonaws.com/runItNow_outline.png?couchbase-example=kotlin-quickstart&source=github)](https://gitpod.io/#https://github.com/couchbase-examples/kotlin-quickstart)
+<!-- [abstract] -->
 
-## Prerequisites
+In this tutorial, you will learn how to connect to a Couchbase Capella cluster to create, read, update, and delete documents and how to write simple parametrized SQL++ queries.
 
+## Prerequisites
 To run this prebuilt project, you will need:
 
-- Follow [Couchbase Installation Options](/tutorial-couchbase-installation-options) for installing the latest Couchbase Database Server Instance (at least Couchbase Server 7)
-- Java SDK v1.8 or higher installed
+- [Couchbase Capella](https://www.couchbase.com/products/capella/) cluster with [travel-sample](https://docs.couchbase.com/kotlin-sdk/current/ref/travel-app-data-model.html) bucket loaded.
+    - To run this tutorial using a self managed Couchbase cluster, please refer to the [appendix](#running-self-managed-couchbase-cluster).
+- [Java SDK v1.8](https://www.oracle.com/java/technologies/downloads/) or higher installed.
```

Comment:
Is it Java SDK 1.8 or Oracle JDK 1.8? Also, is there no dependency on Kotlin?

Reply:
Corrected it to JDK and provided the correct link. The kotlin app needs the JVM, all other dependencies are there in the gradle file.